packages/errors/src/errors.ts

```diff
@@ -75,9 +75,17 @@ export const AllCypressErrors = {
 
         ${fmt.stackTrace(arg1)}`
   },
-  VIDEO_POST_PROCESSING_FAILED: (arg1: Error) => {
+  VIDEO_CAPTURE_FAILED: (arg1: Error) => {
     return errTemplate`\
-        Warning: We failed processing this video.
+        Warning: We failed capturing of this video.
```

Comment:
```suggestion
        Warning: We failed capturing this video.
```

Reply:
addressed in [4191be4](https://github.com/cypress-io/cypress/pull/26800/commits/4191be43ff8d7569a28e598df5bab4ef395cc786)